test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
Nice one 😅 but this file isn't supposed to be here, that's why you were asked to include it in the `.gitignore`, it is mentioned in your `.gitignore` indeed but it's still on the repo. My guess is that it was pushed on GitHub before you created the `.gitignore`. One way to handle this could be deleting the file 📄 on GitHub and trying to recreate it locally to check if it will be pushed. Lemme know if that works 😉